### DISCUSSION TITLE
[General]: Doc for migration from one version to new one

### DISCUSSION THREAD

**Discussion by multivoltage** [davidjerleke/embla-carousel#800] [General]
### Summary

I just update the lib form `7.x` to `8.0.0`

I have a lot of carousel with `align = "center"`.
After updating to v8 all my carousel render like align is `"left"` even if I set `"center"`.
After some hours I saw `"containScroll"`. The docs say that it override `"align"` option in some case.

Then I added containScroll = false and my carousel worked.
Can you consider to add a page dedicated for a migration?

### If applicable, which variants of Embla Carousel are relevant to this conversation?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@multivoltage thanks for your suggestion. The `containScroll` default value change is listed here in the `v8` [release notes](https://github.com/davidjerleke/embla-carousel/releases/tag/v8.0.0) under the heading `What's Changed`:

<img width="700" alt="containScroll" src="https://github.com/davidjerleke/embla-carousel/assets/11529148/1a50bb17-79b5-4dac-827c-78c1aab6bac4">  <br/><br/>


I have no problems with writing a migration guide that's easier to digest for each major release, but my workload is still too big. I need more contributors to help me out with improving the documentation and answering simple questions devs using this library ask here under the [discussions](https://github.com/davidjerleke/embla-carousel/discussions) tab to free up my time. This project isn't paying any of my bills and I can't spend all my spare time (which isn't much to start with) on this project. Me myself don't need any documentation. I know Embla in and out because I do 99% of all work in this project.

So feel free to contribute!

Cheers,
David

**Comment by multivoltage** [comment]
and you did a great job :).
I'm so sorry there is no company who send you some cash :).

Thanks for answer. My fault because I read the release note but for some reason not all rows ;(.
